src/seeders/20191209122959-users.js

```diff
@@ -0,0 +1,49 @@
+import hashPassword from '../helpers/hashPassword';
+
+module.exports = {
+  up: (queryInterface) => queryInterface.bulkInsert('Users', [{
+    firstName: 'victor',
```

Comment:
Do not hard code the value here. Instead, use faker.js

Reply:
How will we know the password set by faker to test with it.